OWNERS

```diff
@@ -1,6 +1,6 @@
 approvers:
-  - richardcase
-  - kubecoins-bot
+- richardcase
+- multipleinheritance
 reviewers
-  - richardcase
-  - kubecoins-bot
+- richardcase
```

Comment:
Cannot parse file: error converting YAML to JSON: yaml: line 5: could not find expected ':'.